### PR TITLE
Add OPUS DOGC dataset

### PR DESCRIPTION
**Pull request by albertvillanova**

### CODE REVIEW

File: datasets/opus_dogc/opus_dogc.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    BUILDER_CONFIG_CLASS = OpusDogcConfig
+    BUILDER_CONFIGS = [OpusDogcConfig(file_format=file_format) for file_format in _FILE_FORMATS]
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "ca": datasets.Value("string"),
+                    "es": datasets.Value("string"),
```
Comment (lhoestq) [inline datasets/opus_dogc/opus_dogc.py:75]: Could you use the `Translation` feature type ?
see [flores](https://github.com/huggingface/datasets/blob/master/datasets/flores/flores.py) for an example of usage

**Review by lhoestq — COMMENTED** [review]
Look very cool, thank you for adding this one.

Usually we use the `Translation` feature type for this kind of dataset, could you do this change please ?

File: datasets/opus_dogc/opus_dogc.py (hunk 1 above)
Comment (albertvillanova) [reply datasets/opus_dogc/opus_dogc.py]: Done, @lhoestq

File: datasets/opus_dogc/README.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+## Dataset Structure
+
+### Data Instances
+
+[More Information Needed]
+
+### Data Fields
+
+[More Information Needed]
```
Comment (yjernite) [inline datasets/opus_dogc/README.md:76]:
```suggestion
A data instance contains the following fields:
- `ca`: the Catalan text
- `en`: the aligned English text
```

**Review by lhoestq — APPROVED** [review]
thanks for doing this change !
LGTM :)

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Pull request merged**